proposals/mips/mip-b38/MIP-B38.md

```diff
@@ -0,0 +1,89 @@
+# MIP-B38: Add USDS Market to Moonwell on Base
+
+## Summary
+
+This proposal seeks to onboard [USDS](https://sky.money/), the stablecoin of Sky
+Protocol, as a new collateral asset on Moonwell’s Base deployment. USDS is a
+decentralized and overcollateralized stablecoin designed to maintain a soft peg
+to the U.S. dollar. With seamless 1:1 conversions to DAI and integration into
+the Sky Protocol ecosystem, USDS provides users with a robust, stable, and
+permissionless asset for lending, borrowing, and liquidity provisioning. Adding
+USDS to Moonwell will enhance stablecoin liquidity, attract new users from the
+Sky Protocol community, and create synergies with Ethereum's broader DeFi
+ecosystem. The stablecoin’s strong foundation, including decentralized
+governance and backing by surplus collateral, ensures reliability and security
+for all users. This proposal aligns with Moonwell’s mission to deliver simple,
+secure, and accessible onchain financial tools. For additional details, please
+review the [USDS token documentation](https://sky-protocol.org/docs) and
+[USDS asset listing forum post](https://forum.moonwell.fi/t/usds-asset-listing/1485/).
+
+## Token Information
+
+- **Name:** USDS
+- **Token Standard:** ERC20
+- **Total Supply:** ∞ (uncapped)
+- **Circulating Supply (Base):** 1,002,682 USDS
```

Comment:
This circulating supply # is incorrect. It is actually 100,840,252.24 according to Basescan. I believe Eli might have mistakenly shown only DEX liquidity